server/app/assets/javascripts/preview.ts

```diff
@@ -34,6 +34,8 @@ class PreviewController {
   private static readonly DEFAULT_ENTITY_TYPE = 'Sample repeated entity type'
   private static readonly DEFAULT_OPTION_TEXT = 'Sample question option'
 
+  private static readonly STATIC_QUESTION_TYPE = 'Static Text'
```

Comment:
**issue (blocking):** TYPE should be TEXT here I believe right?

Reply:
I put type because it was coming from the question types, but updated to text since I think that makes sense too.

Reply:
Text/Type looks here to indicate what data item is being matched and not the broader source/file of that info.